qiskit_experiments/framework/base_experiment.py

```diff
@@ -264,7 +266,31 @@ def run(
         # Generate and transpile circuits
         transpile_opts = copy.copy(experiment.transpile_options.__dict__)
         transpile_opts["initial_layout"] = list(experiment.physical_qubits)
-        circuits = transpile(experiment.circuits(), experiment.backend, **transpile_opts)
+
+        circuits = experiment.circuits()
+
+        # Scheduling parameters
+        if (
+            experiment.backend
```

Comment:
This check is redundant
https://github.com/Qiskit/qiskit-experiments/pull/543/files#diff-0950387eb25e169c9513ea0dacf508dd529698cea5290266ac4ddea5fe58236dL253